src/finland/finlandUtils.ts

```diff
@@ -71,16 +76,28 @@ export const readableNames: Record<string, string> = {
   fiction: "Fiktio",
```

Comment:
"Kirjallisuus" is usually the term used for "Fiction" by the libraries.

In general, all terms in Finnish will have to be replaced with English as source and then passed to the localization.